doc/sphinxext/numpydoc/tests/test_docscrape.py

```diff
@@ -164,6 +179,36 @@ def test_returns():
     assert desc[0].startswith('This is not a real')
     assert desc[-1].endswith('anonymous return values.')
 
+def test_yields():
```

Comment:
Doesn't look like we maintain the tests on our end as there are 4 failing on master, but figured it was worth adding these anyway. Adding the `numpydoc` directory to `PYTHONPATH` and running `pytest tests/test_docscrape.py -k "yield"` yields 3 passes here